bitnami/concourse/templates/web/deployment.yaml

```diff
@@ -293,6 +293,40 @@ spec:
               value: {{ include "concourse.database.port" . }}
             - name: CONCOURSE_POSTGRES_USER
               value: {{ include "concourse.database.user" . }}
+            {{- if .Values.web.conjur.enabled }}
+            - name: CONCOURSE_CONJUR_APPLIANCE_URL
+              value: {{ .Values.web.conjur.applianceUrl | quote }}
+            - name: CONCOURSE_CONJUR_ACCOUNT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "concourse.web.fullname" . }}
+                  key: conjur_account
+            {{- if .Values.secrets.conjurCACert }}
+            - name: CONCOURSE_CONJUR_CERT_FILE
+              value: "/bitnami/concourse/conjur-keys/ca.cert"
+            {{- end }}
+            - name: CONCOURSE_CONJUR_AUTHN_LOGIN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "concourse.web.fullname" . }}
+                  key: conjur_authn_login
+            - name: CONCOURSE_CONJUR_AUTHN_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "concourse.web.fullname" . }}
+                  key: conjur_authn_api_key
+            - name: CONCOURSE_CONJUR_AUTHN_TOKEN_FILE
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "concourse.web.fullname" . }}
```

Comment:
According to the values.yaml file, you cannot use both of them.

Reply:
Only setting Token File is Api Key is empty.